docs/quality-assurance/test-strategy.md

```diff
@@ -36,6 +36,13 @@ Teams should adhere to this strategy when determining their team approach to tes
 * Non-functional requirements and testing should be considered alongside other forms of testing
 * Determining requirements and expected results will involve the architecture and product manager communities
 
+### Safety
```

Comment:
Is it too early to add a line about how to do it? Bowtie?

Reply:
At the moment, will need to add a seperate page once and link it once I have finsihed it.